beacon-chain/rpc/validator/server.go

```diff
@@ -90,6 +90,7 @@ func (vs *Server) WaitForActivation(req *ethpb.ValidatorActivationRequest, strea
 
 	for {
 		select {
+		// Why is this a hard-coded 6?
 		case <-time.After(6 * time.Second):
```

Comment:
hmm thats no good, i think thats the old seconds per slot value

Reply:
+ 1 lets use a param from config